analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -79,6 +79,9 @@
     <Content Include="ResourceTests\**\*">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="..\..\..\README.md" Link="Common\Resources\README.md">
```

Comment:
This should not be needed. Let's add 
```
public TestContext.TestContext { get; set; }
```
and compute the path as relative to `TestContext.DeploymentDirectory`